festipals/src/components/SchedulePage.js

```diff
@@ -16,6 +17,7 @@ export default class SchedulePage extends Component {
 
     return (
       <div>
+      <PageHeader icon="fa fa-calendar" title="Schedule" description="(More information about your schedule)"/>
```

Comment:
I talked wih Petra today, and we thought that we shouldn't add the PageHeader on the SchedulePage, because it takes up about half the screen on a mobile and will therefore block the user from fast access to his/her schedule. It's not there in the mock-up either.